### PR TITLE
working on shell

### PR DESCRIPTION
**Pull request by khairanabila**
``
shell
 - [x] config - on progress
 - [x] data 
 - [x] event
 - [x] exec
 - [x] header
 - [x] input
 - [x] prompt
 - [x] init

### CODE REVIEW

**Comment by slowy07** [comment]
``5b07eabeaa53e4ec5167760279763488cba893e1`` @khairanabila ??

File: software/header.asm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+%include "kernel/header.asm"
+
+[bits 64]
+
+[default rel]
```
Comment (slowy07) [inline software/header.asm:5]: relative addressing ?

Comment (slowy07) [inline software/header.asm:5]: relative addressing ?

File: software/header.asm [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+[org SOFTWARE_BASE_address]
```
Comment (slowy07) [inline software/header.asm:7]: logic mem ?
use comment 
```suggestion
; location  of the program code in logic memory
[org SOFTWARE_BASE_address]
```

**Review by slowy07 — COMMENTED** [review]
check more @khairanabila 😛

File: software/header.asm (hunk 1 above)
Comment (khairanabila) [reply software/header.asm:5]: finish :stuck_out_tongue_closed_eyes:

**Comment by khairanabila** [comment]
> commit zero bootloader : `5b07eabeaa53e4ec5167760279763488cba893e1` @khairanabila ??

forget to pull 😂

**Pull request merged**